.github/workflows/push-master.yml

```diff
@@ -40,16 +40,36 @@ jobs:
       - uses: actions/checkout@v1
         with:
           token: ${{ secrets.GH_PAT }}
+
+      - name: Prepare for app notarization (macOS)
+          if: startsWith(matrix.os, 'macos')
+          # Import Apple API key for app notarization on macOS
+          run: |
+            mkdir -p ~/private_keys/
+            echo '${{ secrets.api_key }}' > ~/private_keys/AuthKey_${{ secrets.api_key_id }}.p8
+
       - name: Build/release Electron app
         uses: samuelmeuli/action-electron-builder@v1
         with:
+          # Base64-encoded code signing certificate for macOS
+          mac_certs: ${{ secrets.mac_certs }}
```

Comment:
Not 100% if actions are clever enough to check the case, but it didn't allow me to add lowercase names, so the names are correct, but it is exposed as `secrets.MAC_CETRS` etc. May be worth-while to just adjust the `secrets.*` to uppercase, just in-case. (Even if it does work and converts)